class code2{
    public static void main(String[] args) {
        catch(exception e){
            System.out.println("exception");
        }
    }
}
/* code2.java:3: error: 'catch' without 'try'
        catch(exception e){
        ^
1 error*/
